src/twisted/python/filepath.py

```diff
@@ -1734,11 +1735,12 @@ def moveTo(self, destination, followLinks=True):
             destination.changed()
 
 
-    @property
+    # Ignore mypy error due to: https://github.com/python/mypy/issues/1362
+    @property  # type: ignore[misc]
     @deprecated(Version('Twisted', 15, 0, 0),
                 "other FilePath methods such as getsize(), "
                 "isdir(), getModificationTime(), etc.")
-    def statinfo(self, value=_SpecialNoValue):
+    def statinfo(self):
```

Comment:
can we just remove statinfo ... as this is deprecated for 5 years?

Reply:
Done